### PR TITLE
feat: changed to SHARP

### PR DESCRIPTION
**Pull request by Grubba27**
This is still work in progress.

--- 
## Todo:
_Add Logo to:_
- [x] Image;
- [x] Gifs(needs testing tho);
- [x] Videos (unaffected)
_Add text to image:_
- [x] Update methods(DSL will not be affected)

### CODE REVIEW

**Review by sibelius — APPROVED** [review]

File: apps/sseramemes/src/image-scripts/addLogoToImage.ts [hunk 1]
```diff
@@ -1,20 +1,51 @@
...
-  ) {
-    logo.opacity(0.5);
+/**
+ * This is a workaround
+ * @deprecated
+ * @param height
+ * @param width
+ */
+const getLogoWithOpacity = (height: number, width: number): string => {
+  if (height < TWITTER_IMAGE_SIZE && width < TWITTER_IMAGE_SIZE)
```
Comment (fersilva16) [inline apps/sseramemes/src/image-scripts/addLogoToImage.ts:12]: This shouldn't be `||`?

**Review by fersilva16 — APPROVED** [review]

File: apps/sseramemes/src/image-scripts/addLogoToImage.ts (hunk 1 above)
Comment (Grubba27) [reply apps/sseramemes/src/image-scripts/addLogoToImage.ts:12]: I thought the same, but previously for some reason I’ve made it with AND and it is running, do you think I should change to an OR?

Comment (fersilva16) [reply apps/sseramemes/src/image-scripts/addLogoToImage.ts:12]: I think so, sometimes I see a meme where the logo overlaps some text, but didn't find an example

Comment (Grubba27) [reply apps/sseramemes/src/image-scripts/addLogoToImage.ts:12]: Will adjust thx for the feedback

**Review by fersilva16 — APPROVED** [review]

**Pull request merged**